arm/implementations/joint_router.hpp

```diff
@@ -1,23 +1,36 @@
 #pragma once
 #include <libhal-util/units.hpp>
 #include <libhal/servo.hpp>
+#include <libhal/accelerometer.hpp>
 
 #include "../platform-implementations/mission_control.hpp"
+#include "../platform-implementations/offset_servo.hpp"
+
+static constexpr float shoulder_gear_ratio = 65/30;
+static constexpr float elbow_gear_ratio = 40/30;
```

Comment:
```suggestion
static constexpr float shoulder_gear_ratio = 65.0f/30.0f;
static constexpr float elbow_gear_ratio = 40.0f/30.0f;
```

Its best to use the correct suffix for floats. 40/30 == 1 if its parsed as an integer.